enclave/src/lib.rs

```diff
@@ -1357,3 +1417,60 @@ fn write_order_to_disk(order: SignedOrder) -> SgxResult<()> {
     }
     Ok(())
 }
+
+pub fn create_in_memory_extrinsic_storage() -> SgxResult<()> {
+    let extrinsic_storage = PendingExtrinsicHelper::create();
```

Comment:
Move all these functions to the extrinsic handler file

Reply:
done